packages/elvis/src/utilities/breakpoints.scss

```diff
@@ -22,3 +22,27 @@ $maxdesktop-min: 1440px;
     }
   }
 }
+
+@mixin lt-mobile() {
+  @media (width <= #{$mobile-max}) {
+    @content;
+  }
+}
+
+@mixin gt-mobile() {
+  @media (width > #{$mobile-max}) {
+    @content;
+  }
+}
+
+@mixin gt-tablet() {
+  @media (width > #{$tablet-max}) {
+    @content;
+  }
+}
+
+@mixin gt-desktop() {
+  @media (width > #{$mindesktop-max}) {
+    @content;
+  }
+}
```

Comment:
I'm not saying we can't use this syntax, but keep in mind it's currently around 90% browser supported 😄 
https://caniuse.com/css-media-range-syntax